modules/ROOT/pages/_partials/commons/common-p2psync-websocket.adoc

```diff
@@ -241,14 +241,16 @@ This option is ignored when dealing with CA certificates.
 === Using Secure Storage
 TLS and its associated keys and certificates might require using secure storage to minimize the chances of a security breach.
 The implementation of this storage differs from platform to platform.
-<<secure-storage-details>> summarizes the secure storage used to store keys and certificates for {param-title}.
+This table summarizes the secure storage used to store keys and certificates.
 
+[tabs#secure-storage-details]
```

Comment:
Need to add CBL-C. CBL-C supports secure key storage on two platforms :

- Apple : Keychain
- Windows : CNG Key Storage Provider

For the other platforms, the application can use external key with callbacks to use with their preferred key storage.

Reply:
Done for Couchbase Lite C only, as it only applies to C.